### ISSUE TITLE
Improve UI of footer

### ISSUE THREAD

**Issue by Naina047** [milligram/milligram#385]
Hey,
I'm a hacktoberfest'23 contributor.
I would like to change the UI of footer.
As this much space doesn't seem to be appealing.
And also change footer background to either purple as theme or greyish as navbar in website .
Please assign this issue to me.

![Screenshot 2023-10-03 172417](https://github.com/milligram/milligram/assets/96466577/d466b0a4-32e2-4ee1-823f-b4a65433ed15)

**Comment by SurajPrakash24** [comment]
Hey I would also like to work on this. Can You plz assign it to me

**Comment by KunjMaheshwari** [comment]
Please I request you to assign this issue to me. I am new to open source. I want to fix this issue in minimum time and with optimized code @Naina047

**Comment by vijaykanwal90** [comment]
This footer need little bit more designing and some design and I have a perfect footer design for it. You can assign me this task

**Comment by sachinchkd** [comment]
Add the horizontal border at top of footer with padding and reduce font size for better look.

**Comment by yatigautam** [comment]
pls assign this issue to me.

**Comment by P-Potdar** [comment]
Yes the footer UI is not appealing I would definitely make it more appealing
